server/controllers/PokerController.js

```diff
@@ -207,6 +208,7 @@ class PokerController {
         let result = table.bootPlayer(user.id, io, room); // boot the player and return any chips they had (not including pot)
         user.chips += result.chips;
         DataAccessLayer.UpdateUser(user);
+        //socket.emit("acountChips", user.chips);
```

Comment:
remove?

Reply:
don't know why, but on my side that line is not commented out